packages/gatsby-source-contentful/src/extend-node-type.js

```diff
@@ -265,6 +272,86 @@ exports.extendNodeType = ({ type }) => {
     return {}
   }
 
+  const getTracedSVG = async (args) =>
+  {
+    const { extname, resolve } = require(`path`)
+    const axios = require(`axios`)
+    const { pathExists, createWriteStream } = require(`fs-extra`)
+    const {
+      traceSVG,
+    } = require(`gatsby-plugin-sharp`)
+
+    const { image, options } = args
+    const { id, file: { url, fileName } } = image
+    const { maxWidth, maxHeight, width, height, resizingBehavior, cropFocus, background } = options
+
+    // Downloading small version of the image with same aspect ratio
+    const assetWidth = maxWidth || width
+    const assetHeight = maxHeight || height
+    let aspectRatio = image.aspectRatio
```

Comment:
I was testing this locally and `image.aspectRatio` was never defined for me - had to use this:
```
let aspectRatio = image.file.details.image.height / image.file.details.image.width
```
what am I missing here?